docs/background/Log_space_statistics/log_variance.rst

```diff
@@ -0,0 +1,27 @@
+*************************
+Outputs from harmonic
+*************************
+
+**harmonic** recovers estimators for :math:`\rho` and :math:`\text{Var}[\rho]`, denoted by :math:`\hat{\rho}` and :math:`\hat{\text{Var}[\rho]}` respectively. In high-dimensional settings these values often are in danger of over- (or under-) flowing, particularly at 32 bit numerical precision. To avoid this practitioners often consider instead estimators for :math:`\log \big ( \rho \big )` upon which they recover estimates of, *e.g.* :math:`\text{Var}[\log \big ( \rho \big )]` *etc.*
```

Comment:
Still does not match notation of paper.  See section 3.2 of paper.  Variable for variance estimator is $\hat{\sigma}^2$.

Remove comment about 32 bit numerical precision (we don't do this so readers might think `harmonic` worse than it is).

No need for big brackets: $\log \big ( \rho \big )$ $\rightarrow$ $\log( \rho)$